frontend/src/routes/(authenticated)/project/[project_code]/+page.ts

```diff
@@ -63,28 +60,26 @@ export async function load(event: PageLoadEvent) {
 						organizations {
 							id
 						}
-            ... on Project @include(if: $userIsMember) {
```

Comment:
alright I've fixed the issue so that members are now filtered (on the backend) if the user is not a member. However I'd like to keep this in place as there's no point in querying the users just for the backend to return an empty list when the user is not a member, although is that causing jwt refresh issues?

Reply:
On the other hand, not requesting the users is sort of meaningless complexity.

Regarding JWT refreshing:
Yes, always selecting the users covers the small edge case of navigating directly to a project that you were added to, but is not yet in your token. That's what my [integration test](https://github.com/sillsdev/languageforge-lexbox/pull/1218/files#diff-6b87f4373687095a81128ab7fdbc2205b1172165a13b96b254134da86e1d8312R91) demonstrates.

It's an edge case, but I felt that handling it was essentially free, so I did. We're not excluding users from the GQL query for performance or security reasons. So, I think it makes sense to leave it.